app/components/windows/go-live/DestinationSwitchers.tsx

```diff
@@ -88,7 +88,7 @@ export class DestinationSwitchers extends TsxComponent<Props> {
             {isPrimary ? (
               <span
                 vTooltip={$t(
-                  'You cannot disable the platform you used to sign in to Streamlabs OBS. Please sign in with a different platform to disable streaming to this destination.',
+                  'You cannot disable the platform you used to sign in to Streamlabs OBS. Please sign in with a different platform to disable streaming to this destinationType.',
```

Comment:
Lol - this looks like a find-replace error.  I assume you didn't mean to change this string.